src/bci_build/package/keadhcp.py

```diff
@@ -0,0 +1,34 @@
+"""KEA DHCP Server BCI container"""
+
+from bci_build.os_version import OsVersion
+from bci_build.package import ApplicationStackContainer
+from bci_build.package.versions import get_pkg_version
+
+_BASE_PODMAN_KEA_CMD = "podman run --replace -it --privileged --network=host"
+_KEA_DHCP4_CONFIG_PATH = "/etc/kea/kea-dhcp4.conf"
+_KEA_DHCP6_CONFIG_PATH = "/etc/kea/kea-dhcp6.conf"
+
+KEA_DHCP_CONTAINERS = []
+
+KEA_DHCP_CONTAINERS.append(
+    ApplicationStackContainer(
+        name="keadhcp",
```

Comment:
I'd suggest to just call it kea, this will yield a more readable registry url `registry.opensuse.org/opensuse/keadhcp:2.6.1` vs `registry.opensuse.org/opensuse/kea:2.6.1`
```suggestion
        name="kea",
```

Reply:
fixed.